ds_judgements_public_ui/sass/includes/_header.scss

```diff
@@ -24,6 +24,7 @@
       display: inline-block;
       position: relative;
       margin-right: calc($spacer__unit / 2);
+      color: $color__white;
```

Comment:
My assumption is this is necessary because the last item isn't a link, unlike every other breadcrumb (and it doesn't make sense to be a link, since 404 pages occur anywhere) -- is that right?

Reply:
Before I added this, the text was black by default, which made it invisible on the header. I might have added it in the wrong place, though (I'm not great at CSS!)